### ISSUE TITLE
[BUG] Playlist doesn't save after reorder

### ISSUE THREAD

**Issue by ForAnAngel731** [FlashpointProject/launcher#436]
**Describe the bug**
There is the ability to rearrange games in a playlist in the List view, but there is no way to save the reordered playlist

**To Reproduce**
1. Go to Games tab and click on Favorites or any of the playlists
2. Change Browser Layout from Grid to List
3. Rearrange order of any number of games by clicking and dragging any game up or down
4. If you click on All Games or any other playlist and then go back to the playlist you reordered, the games are in the original order again.

**Expected behavior**
Reordering games in a playlist should be saved automatically

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Flashpoint Version: Flashpoint 13 Infinity - Dart Frog

**Additional context**
It would make sense if the All Games and Preloaded playlists are not able to be rearranged or altered but the Favorites playlist is created the user so they should be able to rearrange the playlist as well as adding and removing games from it.

**Comment by dot-mike** [comment]
Not reproducible on latest development branch: https://github.com/FlashpointProject/launcher/pull/429

Also I could not reproduce this issue on current release Flashpoint 13 Infinity - Dart Frog.

**Comment by ForAnAngel731** [comment]
I verified the version I'm using and it is the latest version and it's still not working for me. Are you saying it saves the new order in a playlist automatically for you? If I click on another playlist or even the same playlist I'm already looking at, it refreshes and undoes any change I made to the order. Is there a setting I need to set or a button I need to press to save the reorder?

**Comment by dot-mike** [comment]
That's correct, ordering works for me. Reach out on Discord in `#help` if you need any assistance on this.

**Comment by ForAnAngel731** [comment]
Someone said that it was already in the dev branch but not in the current release. So I'm still confused as to how it's working for you with the current release.
